api/types/sortby.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+import (
+	"strings"
+)
+
+func GetSortByFromString(sortStr string) SortBy {
```

Comment:
Could you add some simple tests for this function while you're at it? Check out [`uri_test.go` in `lib/teleterm`](https://github.com/gravitational/teleport/blob/c5c2b5d98b33ce99047a0a678aa6a739a56b0029/lib/teleterm/api/uri/uri_test.go).